### DISCUSSION TITLE
Grafana Faro Version Support Question.

### DISCUSSION THREAD

**Discussion by AbdulTur** [grafana/faro-web-sdk#395] [Q&A] [answered]
Hey, I am wondering if Grafana faro for NPM/CDN supports projects built on node 8 and angularJS?

**Comment by codecapitano** [comment, marked as answer]
Hi @AbdulTur 
Faro aims to be always compatible with current, supported nod lts versions—currenty 18.

That beeing said, using the Faro npm version and building it on node8 will not work.

But using the CDN version should work because you add it to the angular app at runtime.

Cheers,
Marco

**Reply by AbdulTur** [reply]
Thank you
